src/components/MovieDetails/MovieDetailsBody/MovieDetailsBody.tsx

```diff
@@ -0,0 +1,60 @@
+import { useAppSelector } from "../../../redux/hooks";
+import { selectMovieId } from "../../../redux/movieSlice";
+import { useGetMovieQuery } from "../../../redux/apiSlice";
+import { Spinner } from "../../common/Spinner/Spinner";
+
+import styles from "./movieDetailsBody.module.css";
+
+function MovieDetailsBody() {
+  const movieId = useAppSelector(selectMovieId);
+  const { data: movie, isFetching, isSuccess } = useGetMovieQuery(movieId);
```

Comment:
You are making an unnecessary request to the server, if it is possible to avoid the request please try to do it.

You already have all movies data, just save selected one